Plugin.php

```diff
@@ -91,6 +93,30 @@ public function boot()
                 ]
             ]);
         });
+
+        Event::listen('pages.menuitem.listTypes', function() {
+            return [
+                'forum-channel' => 'winter.forum::lang.menuitem.forum_channel',
+                'all-forum-channels' => 'winter.forum::lang.menuitem.all_forum_channels',
+                'all-forum-topics' => 'winter.forum::lang.menuitem.all_forum_topics',
+            ];
+        });
+
+        Event::listen('pages.menuitem.getTypeInfo', function($type) {
+            if ($type == 'forum-channel' || $type == 'all-forum-channels') {
+                return Channel::getMenuTypeInfo($type);
+            } elseif ($type == 'forum-topic' || $type == 'all-forum-topics') {
+                return Topic::getMenuTypeInfo($type);
+            }
```

Comment:
@RomainMazB as a rule, we should use strict comparisons (`===`) in most cases. It allows us to uncover subtle bugs a lot easier.

Also, I didn't see `forum-topic` as an applicable option in the `pages.menuitem.listTypes` event?

```suggestion
            if ($type === 'forum-channel' || $type === 'all-forum-channels') {
                return Channel::getMenuTypeInfo($type);
            } elseif ($type === 'all-forum-topics') {
                return Topic::getMenuTypeInfo($type);
            }
```

Reply:
You right, nice catch!
I first wanted to add it but then I said to myself that I don't see any use case where we could add only one topic in the sitemap.